### ISSUE TITLE
JavaCV 1.0 issue "couldn't find "libjniopencv_core.so", with openCV built opencv-3.0.0-1.0.1-20150718.074640-2

### ISSUE THREAD

**Issue by xiaos** [bytedeco/javacv#250]
I am using the latest FFMPEG and opencv libs, it'w working find on Samsung phones, but it throws errors on mi 4i phone.

Can you please help ?

```
10-12 10:19:58.312 17368-17368/com.apps.test E/art: dlopen("/system/lib64/libdl.so", RTLD_LAZY) failed: dlopen failed: cannot locate symbol "__cxa_finalize" referenced by "/system/lib64/libdl.so"...

10-12 10:19:58.312 17368-17368/com.apps.test E/art: dlopen("/system/lib64/libdl.so", RTLD_LAZY) failed: dlopen failed: cannot locate symbol "__cxa_finalize" referenced by "/system/lib64/libdl.so"...
10-12 10:19:58.362 17368-17368/com.apps.test D/AndroidRuntime: Shutting down VM
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime: FATAL EXCEPTION: main
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime: Process: com.apps.test, PID: 17368
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.apps.test-1/base.apk"],nativeLibraryDirectories=[/data/app/com.apps.test-1/lib/arm64, /vendor/lib64, /system/lib64]]] couldn't find "libjniopencv_core.so"
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.Runtime.loadLibrary(Runtime.java:366)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.System.loadLibrary(System.java:989)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:550)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.load(Loader.java:415)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.load(Loader.java:358)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.opencv_core.<clinit>(opencv_core.java:10)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.Class.classForName(Native Method)



10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:  Caused by: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.apps.test-1/base.apk"],nativeLibraryDirectories=[/data/app/com.apps.test-1/lib/arm64, /vendor/lib64, /system/lib64]]] couldn't find "libopencv_imgproc.so"
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.Runtime.loadLibrary(Runtime.java:366)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.System.loadLibrary(System.java:989)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:550)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.load(Loader.java:406)
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.load(Loader.java:358) 
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.opencv_core.<clinit>(opencv_core.java:10) 
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.Class.classForName(Native Method) 
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at java.lang.Class.forName(Class.java:308) 
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.load(Loader.java:390) 
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.Loader.load(Loader.java:358) 
10-12 10:19:58.372 17368-17368/com.apps.test E/AndroidRuntime:     at org.bytedeco.javacpp.opencv_imgproc$CvFont.<clinit>(opencv_imgproc.java:2315) 
```

**Comment by saudet** [comment]
If you need 64-bit binaries, you're going to need to provide it with 64-bit binaries. Please let me know if you would like to work on the binaries for `arm64`, thank you.

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacpp-presets/issues/52

**Comment by xiaos** [comment]
hi @saudet , Thank you so much for your reply.

Yes, I would like to support more devices, so if I need support amr64, i need prepare two set of .so libs for both arm32 and arm 64?

**Comment by saudet** [comment]
No, 64-bit ARM devices support 32-bit binaries. But it looks like you want to use 64-bit binaries, so you are going to need 64-bit in that case.

**Comment by xiaos** [comment]
Hi @saudet ,

Yes, I need 64-bit binaries, and how to get it? and I think this year more and more 64-bit arm devices, we should support 64-bit arm, right?

**Comment by saudet** [comment]
Yes, we should. To do that, we would need to add `android-arm64` entries to all the `cppbuild.sh` files. We can start by copying the `android-arm` entries and do what needs to be done to make the builds pass for 64-bit. If this sounds like something you would like to do, let me know if you encounter any problems, thank you!

**Comment by xiaos** [comment]
@saudet   Hi, thank you, and I will try to do that!

**Comment by hs9021401** [comment]
@xiaos Hi. Have you solved this problem. I faced the same issue.

**Comment by magnusjjj** [comment]
Bump here too :). Might take a look at it.

**Comment by saudet** [comment]
@magnusjjj Great, thanks! :+1:

**Comment by magnusjjj** [comment]
Nope. All my build tools spontaneously combusted (aka, android studio halfway installed an update, gradle is screaming profanities, and somehow my virtual machine got messed up. Not my... week :P). Think i am giving up for the moment. Sorry <3

**Comment by saudet** [comment]
64-bit builds for Android (as the "android-arm64" and "android-x86_64" platforms) are now available, so please give it a try! Binaries for 1.4.1-SNAPSHOT can be downloaded from the snapshot repository: http://bytedeco.org/builds/
